docs/source/profiler.rst

```diff
@@ -464,12 +464,14 @@ Below is an breakdown of all the options.
       
       * is_case_sensitive - (Boolean)
       * stop_words - (List of Strings)
+      * top_k_chars - (Int) Apply to the output of vocab count when profiling
```

Comment:
I think these descriptions are a little vague, can this be improved? Also, unrelated, can you add descriptions for the 2 options above?

Reply:
added, @gme5078